Topic5Mysql/src/DataBaseTest.java

```diff
@@ -0,0 +1,74 @@
+import java.net.ConnectException;
+import java.sql.ResultSet;
+
+public class DataBaseTest {
+
+	public static void main(String[] args) throws Exception {
+
+		// Connection
+
+		DataBaseConnection dataBaseConnection = new DataBaseConnection();
+		dataBaseConnection.connect();
+
+		// Test exercise 4
+
+		System.out.println(" ");
+		System.out.println(" ---Excercise 4--- ");
+		System.out.println(" ");
+
+		DataBaseManager dataBaseManager = new DataBaseManager();
+
+		ResultSet resultSet1;
+		ResultSet resultSet2;
+
+		int course_id = 2;
+
+		String query1 = "SELECT course.course_name, teacher.last_name, teacher.first_name " + "FROM course, teacher " + "WHERE course_id ="

```

Comment:
What you did is an alternative (and old way to do an inner join), it is not bad, but as you can see in the url below isn't recommended
http://sqlblog.com/blogs/aaron_bertrand/archive/2009/10/08/bad-habits-to-kick-using-old-style-joins.aspx